geotrellis/maml
# Add STRTA and migrate to CircleCI

## Overview

- Add `cibuild` and `test` STRTA 
- Migrate from Travis to CircleCI 
- Update [paulp/sbt-extras](https://github.com/paulp/sbt-extras) to support OpenJDK 11
- Add [Coursier](https://get-coursier.io)



## Notes

I had tried to add OpenJDK 11 to the build matrix, but was running into [compatibility issues](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html#jdk-11-compatibility-notes). I think we should split this out into a separate issue. 

Scala 2.12.8 builds are also failing. I took a pass but I'm not immediately sure what's up. I'm inclined to leave Scala 2.12.8 in the build matrix, as it seems more important than supporting OpenJDK 11, but I'm not sure what the path forward should be. 

## Testing Instructions

See the CI workflow: https://circleci.com/workflow-run/ea13448f-59ae-4414-9b12-abe5252104a5
